test/node-api/test_general/test_general.c

```diff
@@ -39,8 +89,9 @@ static napi_value GetFilename(napi_env env, napi_callback_info info) {
 
 static napi_value Init(napi_env env, napi_value exports) {
   napi_property_descriptor descriptors[] = {
-    DECLARE_NODE_API_PROPERTY("testGetNodeVersion", testGetNodeVersion),
-    DECLARE_NODE_API_GETTER("filename", GetFilename),
+      DECLARE_NODE_API_PROPERTY("testGetNodeVersion", testGetNodeVersion),
+      DECLARE_NODE_API_PROPERTY("bufferFromArrayBuffer", bufferFromArrayBuffer),
```

Comment:
There must be test JS changes to call the new test method. Should it be named testCreateBufferFromArrayBuffer?